.github/workflows/dev.yml

```diff
@@ -37,7 +37,26 @@ jobs:
   toolbox-tests:
     uses: ./.github/workflows/reusable-toolbox-tests.yml
 
-  aura-tests:
+  aura-free-tests:
+    if: github.event.pull_request.merged == true
```

Comment:
question: Is this "PR got merged" check really required for the `dev` workflow?

Reply:
Great spot, good old bit of copy and paste. Fixed!